web-share/resources/manual-helper.js

```diff
@@ -0,0 +1,71 @@
+// Sets up the page for running manual tests. Automatically creates the
+// instructions (based on the parameters) and the share button.
+function setupManualShareTest(
+    user_should_cancel, expected_title, expected_text, expected_url) {
+  let div = document.createElement('div');
```

Comment:
Nit: use "const" unless you are planning on reassigning the variable. Applies globally.